install/linux/usr/share/odemis/sim/sparc2-sim.odm.yaml

```diff
@@ -240,12 +227,15 @@ SPARC2: {
     },
     affects: ["Camera", "Spectral Camera", "Spectrometer IR"],
 }
-
 "Optical Path Properties": {
     class: static.OpticalLens,
     role: lens,
     # Standard mirror config
     init: {
+       configurations: {
```

Comment:
Only these lines are to be added. All the other changes in this file should be reverted.